packages/cozy-client/src/CozyClient.js

```diff
@@ -1761,13 +1761,16 @@ instantiation of the client.`
    * @returns {Promise<void>}
    */
   async loadInstanceOptionsFromStack() {
-    const { data } = await this.query(
-      Q('io.cozy.settings').getById('io.cozy.settings.capabilities')
-    )
+    const results = await Promise.all([
+      this.query(
+        Q('io.cozy.settings').getById('io.cozy.settings.capabilities')
+      ),
+      this.query(Q('io.cozy.settings').getById('io.cozy.settings.instance'))
+    ])
 
-    const { data: instanceData } = await this.query(
-      Q('io.cozy.settings').getById('io.cozy.settings.instance')
-    )
+    const { data } = results[0]
+
+    const { data: instanceData } = results[1]
```

Comment:
It might be clearer and more consistent to have: 
```
const { data: capabilitiesData } = results[0]
const { data: instanceData } = results[1]
```

Reply:
[Done](https://github.com/cozy/cozy-client/compare/f503907fa6bfdf87169ff30a4875f649d113b048..084258f062b4c810c59927176c3e25a0d5a8124d)